scripts/metadata-export-search.md

```diff
@@ -0,0 +1,41 @@
+# Metadata-Export-Search Script
```

Comment:
add a link back to the list of the scripts (something similar is done in each individual endpoint file toward the endpoint.md list)